Dockerfile_browser_tools.dockerfile

```diff
@@ -9,14 +9,13 @@ RUN npx puppeteer browsers install chrome
 
 # # Tell Puppeteer to skip installing Chromium. We'll be using the installed package.
 ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
-ENV PUPPETEER_EXECUTABLE_PATH=/home/circleci/.cache/puppeteer/chrome/linux-121.0.6167.85/chrome-linux64/chrome
 
 # Chrome version 121.0.6167.185 is mapped to Puppeteer version 21.9.0, as per documentation -> https://pptr.dev/chromium-support
 RUN yarn add puppeteer@21.9.0
 
 # Install PDFTK
-RUN sudo add-apt-repository --yes ppa:malteworld/ppa || true
-RUN sudo apt update --allow-unauthenticated || true
-RUN sudo apt install pdftk --allow-unauthenticated || true
+RUN sudo apt update --allow-unauthenticated
```

Comment:
I remember that we needed the `|| true` here, from several rounds of commits.

Reply:
the `|| true` is only required for the line:
RUN sudo apt update --allow-unauthenticated
It can be removed from all other lines and the image builds successfully
it is needed because it always fails and the '|| true' part allows it to continue
I think the update line requires it because after the new repo is added it has an issue updating, but as Patrick pointed out, standard practice is to run the update command before installing anything, so moving the update to the start allows it to run and properly